src/components/FTextContent.vue

```diff
@@ -1,3 +1,5 @@
+<!-- eslint-disable vue/no-v-html -->
+<!-- An XSS attack is not a risk here -->
```

Comment:
Not true...it is an open risk (because when using FTextContent we will pass html directly to the component), but it is a risk that we intentionally make and accept, and the responsibility is explicitly on the side of the one using FTextContent 😉 
```suggestion
<!-- The purpose of FTextContent is to render any HTML content -->
```